docs/RFCS/20211106_multitenant_cluster_settings.md

```diff
@@ -215,11 +215,11 @@ following guidelines:
 
  - control settings relevant to tenant-specific internal implementation (like
    tenant throttling) that we want to be able to control per-tenant should be
-   `system`.
+   `tenant-ro`.
 
  - when in doubt the first choice to consider should be `tenant-ro`.
```

Comment:
apologies for this off-topic drive-by comment, that is also late and should have been raised on the RFC (so feel free to just hit "Resolve Discussion" here if this is too off-topic for this pr), but should't this "when in doubt the first choice" be `tenant-rw`, not `tenant-ro`?  like, by _default_ a tenant should have the same controls and capabilities as any other user over the behavior of their cluster, and we should only reach for the restrictive ones when we need them, right? particularly in the the world where we can override from host with a value at runtime if we need to with no code change, even if it is tenant-rw in the code?